### PR TITLE
revised login ui for the business website

### PR DESCRIPTION
**Pull request by HiltonLuu**

### CODE REVIEW

File: next/src/app/sign-in/[[...sign-in]]/page.tsx [hunk 1]
```diff
@@ -1,10 +1,34 @@
...
+    <div className="flex justify-center items-center min-h-screen gap-20">
+      <div className="flex flex-col items-center gap-10">
+        <div className="flex items-center gap-5">
+          <Image
+            src="/assets/Logo.png"
+            width={80}
+            height={80}
+            alt="Picture of LocaLoyalty Logo"
+          />
+          <p className="text-5xl">LocaLoyalty</p>
```
Comment (J-essicaTran) [inline next/src/app/sign-in/[[...sign-in]]/page.tsx]: Looks great!

I'd recommend adding `font-semibold` to `<p className="text-5xl">LocaLoyalty</p>` in both the sign-in and sign-up page.

File: next/public/assets/LHS.png
Comment (J-essicaTran) [inline next/public/assets/LHS.png:1]: As for the illustrations, they are a little blurry. So maybe try exporting them at a larger size to see if that helps.

**Review by J-essicaTran — CHANGES_REQUESTED** [review]
Looks great!

I'd recommend adding "font-semibold" to < p className="text-5xl">LocaLoyalty< /p > in both the sign-in and sign-up page.

As for the illustrations, they are a little blurry. So maybe try exporting them at a larger size to see if that helps.

**Review by HiltonLuu — COMMENTED** [review]
added the font change in subsequent commit

**Review by J-essicaTran — APPROVED** [review]

**Review by HiltonLuu — COMMENTED** [review]
confirming changes

**Pull request merged**